app/models/foreman_fog_proxmox/proxmox_volumes.rb

```diff
@@ -59,9 +59,9 @@ def update_cdrom(vm, disk, volume_attributes)
     end
 
     def extend_volume(vm, id, diff_size)
-      extension = '+' + (diff_size / GIGA).to_s + 'G'
-      logger.info(format(_('vm %<vmid>s extend volume %<volume_id>s to %<extension>s'), vmid: vm.identity,
-volume_id: id, extension: extension))
+      extension = '+' + diff_size.to_s + 'G'
+      logger.info(format(_('vm %<vmid>s extend volume %<volume_id>s to %<extension>s'),
```

Comment:
I realize it's not part of this PR, logger statements shouldn't be translated.

Reply:
I will see if theforeman-rubocop rules accept this style